### ISSUE TITLE
faulty dependency in 3.0.0-rc1 pom

### ISSUE THREAD

**Issue by janklecka0815** [otto-de/jlineup#10]
In the pom of 3.0.0-rc1 jlineup-core is not properly referenced.
Is:
```xml
<dependency>
    <groupId>de.otto</groupId>
    <artifactId>core</artifactId>
    <version>3.0.0-rc1</version>
    <scope>compile</scope>
</dependency>
```
Should:
```xml
<dependency>
    <groupId>de.otto</groupId>
    <artifactId>jlineup-core</artifactId>
    <version>3.0.0-rc1</version>
    <scope>compile</scope>
</dependency>
```

This leads to problems if you want to use jlineup in gradle:
Execution failed for task ':jlineupBefore'.
> Could not resolve all files for configuration ':runtime'.
    > Could not find de.otto:core:3.0.0-rc1.
      Searched in the following locations:
          https://repo1.maven.org/maven2/de/otto/core/3.0.0-rc1/core-3.0.0-rc1.pom
          https://repo1.maven.org/maven2/de/otto/core/3.0.0-rc1/core-3.0.0-rc1.jar
      Required by:
         project : > de.otto:jlineup:3.0.0-rc1

**Comment by MediaMarco** [comment]
Should be fixed in 3.0.0-rc2

**Comment by janklecka0815** [comment]
Works, thanks :)
